charts/rasa-x/templates/rasa-config-files-configmap.yaml

```diff
@@ -48,6 +48,10 @@ data:
       {{- end }}
       {{- end }}
     {{- end }}
+    {{- if $.Values.kafka.enabled }}
```

Comment:
Can we add `fail` to the `NOTES.txt` if `rabbitmq.enabled=true` and `kafka.enabled=true` - something like we have in the rasa helm chart - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L42-L45